### PR TITLE
Adding RAG Fusion

### PR DESCRIPTION
**Pull request by Nishant-Kumar-2002**
Addition of RAG Fusion
- by Nishant

### CODE REVIEW

**Comment by Nishant-Kumar-2002** [comment]
Done.

**Comment by PrashantDixit0** [comment]
I'll look into the notebook and provide comments 👍

File: examples/RAG Fusion/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## RAG Fusion - The New Star of Search Technology
+
+### Overview
+This repository contains code and implementation for RAG Fusion which is the enhanced version of RAG
+and the future of RAG Technology for the improvement of RAG. This technology is still in process 
+for getting the best out of it.
```
Comment (AyushExel) [inline examples/RAG Fusion/README.md:6]: This doesn't make a lot of sense. Maybe just use the features section for the overview?

**Comment by Nishant-Kumar-2002** [comment]
All the changes have been done.

**Comment by AyushExel** [comment]
Add the example to this table in the main readme

Screenshot from 2024-01-17 19-31-33

**Comment by Nishant-Kumar-2002** [comment]
Done.

**Comment by AyushExel** [comment]
Screenshot from 2024-01-17 19-55-44
Thanks...one last thing..Can you install all these in a single pip install line ?

**Comment by AyushExel** [comment]
Thanks!!

**Review by AyushExel — APPROVED** [review]

**Pull request merged**